src/errors.ts

```diff
@@ -27,7 +27,15 @@ export async function handleError(ctx: Context, error: Error, labels?: Labels) {
 
 	const status = (error as HTTPError).status ?? 500;
 	const message = error.message || 'Server Error';
-	ctx.wshimLogger.log(message);
+
+	ctx.wshimLogger.error({
+		message,
+		status,
+		prefix: ctx.prefix,
+		...(ctx.key_id && { key_id: ctx.key_id }),
+		...(labels?.path && { path: labels.path }),
+	});
```

Comment:
slightly longer but likely easier to read

```suggestion
  const logEntry = { error, status, prefix: ctx.prefix }
  if (ctx.key_id) {
    logEntry.key_id = ctx.key_id
  }
  if (labels?.path) {
    logEntry.path = labels.path
  }
  ctx.wshimLogger.log(logEntry);
	ctx.wshimLogger.error({
		message,
		status,
		prefix: ctx.prefix,
		...(ctx.key_id && { key_id: ctx.key_id }),
		...(labels?.path && { path: labels.path }),
	});
```